### DISCUSSION TITLE
Retaining the person image against the logbook entry

### DISCUSSION THREAD

**Discussion by logicaldeveloper** [blakeblackshear/frigate#9631] [General]
Is there a way to retain the image of camera_name_person in the log book for previous events
i.e. https://github.com/9pm if an event detected and a person image updated in camera_name_person
and at 10PM if another event with another person detected, then this image gets updated in the camera_name_person

when we see the logbook to see past events the 9PM event entry will also show 10PM event entry image. is there any way to persist the corresponding image against the corresponding event

**Comment by NickM-27** [comment]
Not in home assistant, it could be done separately

**Reply by logicaldeveloper** [reply]
could you let me know how to do it. i am very new to HA, jus exploring for past one month. Earlier used inside AWS but i got charged as i crossed the network limit so now am using HAOS in my desktop.
As of now i've enabled snapshots so if i go inside the Media Folder, i could see the old images
but it would be awesome if the log book also shows the same image, may be if we make this images appear in log book (not as an event, but jus with the time when the images stored may those 2 entries will be next to each other, we can related
